### PR TITLE
it works first try were the best

### PR DESCRIPTION
**Pull request by MBorchert**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -161,10 +161,10 @@ public static final class AutoConstants {
                                                            // but spinning fast is not particularly useful or driver
                                                            // friendly
 
-    public static final double kPXController = 2;
-    public static final double kDxController = 0.00005;
+    public static final double kPXController = 1.5;
+    public static final double kDxController = 0.000075;
```
Comment (stolenricecakes) [inline src/main/java/frc/robot/Constants.java]: Were these numbers tuned to protobot?  Suggest not carrying this forward as comp is likely to be different.

File: src/main/java/frc/robot/subsystems/PoseEstimationSubsystem.java [hunk 2]
```diff
@@ -88,7 +88,7 @@ public Pose2d getPose() {
     private void updateEstimation() {
         estimator.update(drivetrain.getGyro(), drivetrain.getModulePositions());
         if (visionSubsystem.seeingAnything()) {
-        //    estimator.addVisionMeasurement(visionSubsystem.getRobotPose(), Timer.getFPGATimestamp());
+           estimator.addVisionMeasurement(visionSubsystem.getRobotPose(), Timer.getFPGATimestamp());
```
Comment (stolenricecakes) [inline src/main/java/frc/robot/subsystems/PoseEstimationSubsystem.java]: This is very risky to uncomment unless we've validated it on compbot, on an actual real field

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -161,10 +161,10 @@ public static final class AutoConstants {
...
     // public static final double kPYController = 0.0000;
-    public static final double kPThetaController = 1.25;
+    public static final double kPThetaController = 2;
```
Comment (stolenricecakes) [inline src/main/java/frc/robot/Constants.java]: Same here... If you want stuff for protobot, then move this into the config

**Review by stolenricecakes — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (khaiwall) [reply src/main/java/frc/robot/Constants.java]: um, well, yes. on protobot. we should change these probbaly

**Pull request closed without merging**